File: C programming/Assignment/Assignment5/type4/prime.c
#include <stdio.h>

int prime(int num);
void main(){
    int num;
    printf("Enter the number:");
    scanf("%d",&num);
    if(prime(num)==0)
        printf("prime number");
    else
        printf("not prime");
}
int prime(int num){
    int i=2,flag=0;

    while(i<num){
        if(num%i ==0){
            flag = 1;
            break;
        }
        i++;
    }
    return flag;
}

File: Java_programming/basic_java/Prime.java
package type4;

public class Prime {

	public static void main(String[] args) {
		int num=7;
//	    printf("Enter the number:");
//	    scanf("%d",&num);
	    if(prime(num)==0)
	        System.out.printf("prime number");
	    else
	    	System.out.printf("not prime");

	}
	
	public static int prime(int num){
	    int i=2,flag=0;

	    while(i<num){
	        if(num%i ==0){
	            flag = 1;
	            break;
	        }
	        i++;
	    }
	    return flag;
	}

}
